.github/workflows/deploy_and_release.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         ARBISCAN_API_KEY: ${{ secrets.ARBISCAN_API_KEY }}
         OPTIMISTIC_ETHERSCAN_API_KEY: ${{ secrets.OPTIMISTIC_ETHERSCAN_API_KEY }}
         ZKSYNC_EXPLORER_API_KEY: ${{ secrets.ZKSYNC_EXPLORER_API_KEY }}
+        BARTIO_BERA_EXPLORER_API_KEY: ${{ secrets.BARTIO_BERA_EXPLORER_API_KEY }}
```

Comment:
Is the key already added in github secrets?

Reply:
not yet